### ISSUE TITLE
Student View link on preview page (Materia LTI w/ Canvas)

### ISSUE THREAD

**Issue by mrash2** [ucfopen/Materia#1343]
In a Canvas assignment with Materia widget, the preview page includes a message at the bottom that links to Student View; however, the link is to the UCF Canvas instance. Is there a quick way to revise or remove the link ourselves, or could this be updated in a future release? At least for us at UTK, we'd rather folks access the Student View button at the top of the page anyway.

**Comment by clpetersonucf** [comment]
I don't believe this link is present in the student view in the latest version(s) of Materia. Please advise if that's not the case.
